apps/institution-service/.env.example

```diff
@@ -0,0 +1,10 @@
+AUTH0_AUDIENCE=
```

Comment:
I don't think these 2 variables are sensitive, and they're required for this service to function

Reply:
I think we should move them out of the env and hardcode them in a variable somewhere

Reply:
Are you trying to say it should be hardcoded everywhere it's used? I could see these variables potentially being different for different environments so it makes sense to me to have them here.